packages/__docs__/webpack.config.js

```diff
@@ -43,7 +43,7 @@ const config = merge(baseConfig, {
   },
   output: {
     path: outputPath,
-    filename: '[name].[contenthash].js'
+    filename: '[name].js'
   },
```

Comment:
This has to be changed because it would generate such files as `globals.[contenthash].js` and it messed up the codepen resource links.